controllers/index.js

```diff
@@ -36,17 +38,48 @@ exports.getArticles = (req, res, next) => {
 exports.getArticleById = (req, res, next) => {
   const { article_id } = req.params;
   if (!/^\d+$/.test(article_id)) {
-    res.status(400).send({ msg: "invalid id" });
+    res.status(400).send({ message: "Invalid ID" });
   } else {
     fetchArticleById(article_id)
-      .then(({ rows }) => {
+      .then((article) => {
+        res.status(200).send({ article });
+      })
+      .catch(next);
+  }
+};
+
+exports.getComments = (req, res, next) => {
+  const { article_id } = req.params;
+  if (!/^\d+$/.test(article_id)) {
+    res.status(400).send({ message: "Invalid ID" });
+  } else {

```

Comment:
This logic does not need to be here as this can be handled by psql, which we can then use to deal with the error.